awesomplete.js

```diff
@@ -279,9 +279,19 @@ _.SORT_BYLENGTH = function (a, b) {
 
 function regEscape(s) { return s.replace(/[-\\^$*+?.()|[\]{}]/g, "\\$&"); }
 
-$$("input.awesomplete").forEach(function (input) {
-	new Awesomplete(input);
-});
+function init() {
+	$$("input.awesomplete").forEach(function (input) {
+		new Awesomplete(input);
+	});
+}
+
+// DOM already loaded?
+if (document.readyState !== 'loading') {
```

Comment:
What about other states, before `loading`? Are we sure they’re not relevant here?